docs/reference/functions.md

```diff
@@ -101,9 +101,35 @@ notMatch | if provided, value must _not_ match this regex | no
         notMatch: ".+\/$"
 ```
 
+## casing
+
+Text must match a certain case, like `camelCase` or `snake_case`. 
+
+<!-- title: functionOptions -->
+
+name | description | required?
+---------|----------|---------
+type | the casing type to match against | yes
+disallowDigits | if not truthy, digits are allowed | no
+
+Available types are: flat, camel, pascal, kebab, cobol, snake, macro.
```

Comment:
```suggestion
Available types are: 

| name   | sample         |
|--------|----------------|
| flat   | verylongname   |
| camel  | veryLongName   |
| pascal | VeryLongName   |
| kebab  | very-long-name |
| cobol  | VERY-LONG-NAME |
| snake  | very_long_name |
| macro  | VERY_LONG_NAME |
```

Reply:
This! @P0lip accept this one and we can merge.

Reply:
This is great. Thank you so much @nulltoken